### ISSUE TITLE
`scrollbar` incorrectly reported as color

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#403]
```css
-ms-overflow-style: scrollbar;
```

gets reported as a color.
